docs/develop/09_polarpath/01_overview.md

```diff
@@ -0,0 +1,91 @@
+---
+id: what-is-polarpath
+title: Overview
+---
+
+Polar path consists of an `asset-swap` pallet for KILT chain runtimes and extrinisics that let parachain-based chain developers to make their native token accessible on the Ethereum network using [the Snowbridge bridge](https://docs.snowbridge.network) between Polkadot and Ethereum.
+
+Polar path provides parachains with a solution for creating an ERC-20 wrapper around their native tokens, enabling a trustless conversion of any existing native parachain token (PARA) into a wrapped parachain token (wPARA) on Ethereum while maintaining the trustlessness guarantees Snowbridge provides.
+
+The pallet enables a configured conversion rate of parachain tokens on the origin chain to an ERC-20 wrapper token on Polkadot's AssetHub.
```

Comment:
False, this is not happening anymore. It's always 1:1.

Reply:
I am not sure what you're referring to? Most of my source material was old, so if you could point me to any more up-to-date content that explains what you mean, that would be appreciated.

Reply:
It's in the pallet docs. But anyway, the pallet only allows 1:1 exchanges. We scrapped the configurable ratio because of rounding issues and also because we are not sure it's something people need. If they do, we'll rethink how to do that.